From 2b23e00c1758d3a78e6862323f69dd7c1cc689b9 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:42 -0400
Subject: [usb] change dev_warn about missing reset-resume to dev_dbg

Message-id: <1410887463-20674-91-git-send-email-dzickus@redhat.com>
Patchwork-id: 95146
O-Subject: [RHEL7 PATCH 090/591] USB: change dev_warn about missing reset-resume to dev_dbg
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 0a56b4fa6844f74976d4bfa4956fa6d606b392ea
Author: Alan Stern <stern@rowland.harvard.edu>
Date:   Fri Oct 18 11:17:21 2013 -0400

    USB: change dev_warn about missing reset-resume to dev_dbg

    This patch changes a dev_warn() call in usbcore to dev_dbg().  It's
    not necessary to warn about drivers missing a reset-resume callback,
    since the reset-resume method is optional.

    Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
    CC: Clemens Ladisch <clemens@ladisch.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/driver.c b/drivers/usb/core/driver.c
index 689433c..47aade2 100644
--- a/drivers/usb/core/driver.c
+++ b/drivers/usb/core/driver.c
@@ -1179,8 +1179,8 @@ static int usb_resume_interface(struct usb_device *udev,
       "reset_resume", status);
   } else {
    intf->needs_binding = 1;
-   dev_warn(&intf->dev, "no %s for driver %s?\n",
-     "reset_resume", driver->name);
+   dev_dbg(&intf->dev, "no reset_resume for driver %s?\n",
+     driver->name);
   }
  } else {
   status = driver->resume(intf);
-- 
1.7.1